assets/js/pages/AdvisoryDetails/AdvisoryDetails.test.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+
+import { faker } from '@faker-js/faker';
+
+import { renderWithRouter as render } from '@lib/test-utils';
+import { advisoryErrataFactory } from '@lib/test-utils/factories';
+
+import AdvisoryDetails from './AdvisoryDetails';
+
+describe('AdvisoryDetails', () => {
+  it('displays a message, when the CVE, packages or fixes section is empty', () => {
```

Comment:
Why do we only test this case, and not `fixes`/`cves`/`packages` being populated also?